src/front/config/mainnet/swapContract.js

```diff
@@ -13,6 +14,8 @@ export default {
   uniswapFactory: '0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f',
   pancakeswapRouter: '0x10ED43C718714eb63d5aA57B78B54704E256024E', // BSC
   pancakeswapFactory: '0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73',
+  carthageRouter: '0x1a45c9f823F64c9360b0c3df269F824cf404f82b', // Candle
+  carthageFactory: '0x5Bb7BAE25728e9e51c25466D2A15FaE97834FD95',
```

Comment:
Can you attach links to this project/code?

Reply:
Yes, it's pretty small for now but it's the first dex on candle. The GitHub is [here](https://github.com/carthagedex).

Reply:
![image](https://user-images.githubusercontent.com/61930014/175873551-ebfb53bb-3c75-4790-bc41-a5918fd41068.png)

Reply:
> ![image](https://user-images.githubusercontent.com/61930014/175873551-ebfb53bb-3c75-4790-bc41-a5918fd41068.png)
> 
> 

Apologies, fixed ! [Here](https://github.com/carthageprotocol) it is again

Reply:
Link to the working project

Reply:
https://dex.CNDLchain.com